src/enums/correctionTypes.ts

```diff
@@ -0,0 +1,5 @@
+export enum CorrectionTypes {
+    NONE = '',
```

Comment:
NONE is not a valid enum value, according to the schema.

Use null for "no value".